vaadin/framework
# Test fixes and tweaks

- Properly init client-side data in GridDataChangeHandlerTest
- Update Chrome version
- Separate and refactor ComboboxMenuBarAutoopenTest test cases to match
test description
- Update screenshot for ResponsiveStylesTest
- Refactor TextAreaEventPropagationTest to better match expected
behavior
- Add wait to GridDetailsClientTest for stability


This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/vaadin/framework/12239)
